lib/rspec/support/encoded_string.rb

```diff
@@ -54,7 +70,7 @@ def to_s
         #     vs "\x80".encode('UTF-8','ASCII-8BIT', undef: :replace, replace: '<undef>')
         #     # => '<undef>'
         #   Encoding::CompatibilityError
-        #    when Enconding.compatbile?(str1, str2) is false
+        #    when Encoding.compatibile?(str1, str2) is nil
```

Comment:
Similarly you've unified this elsewhere so this needs indenting an additional space and then the text beneath indenting to 4 spaces.